argo_rollouts/enable-support-for-namespace-scoped-argo-rollouts-installation.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="enable-support-for-namespace-scoped-argo-rollouts-installation"]
+= Enabling support for a namespace-scoped Argo Rollouts installation
+:context: enable-support-for-namespace-scoped-argo-rollouts-installation
+
+toc::[]
+{gitops-title} enables support for two modes of Argo Rollouts installations:
+
+* *Cluster-scoped installation* (default): The Argo Rollouts custom resources (CRs) defined in any namespace are reconciled by the Argo Rollouts instance. As a result, you can use Argo Rollouts CR across any namespace on the cluster.
+* *Namespace-scoped installation*: The Argo Rollouts instance is installed in a specific namespace and only handles an Argo Rollouts CR within the same namespace. This installation mode includes the following benefits:
+** This mode does not require cluster-wide `ClusterRole` or `ClusterRoleBinding` permissions. You can install and use Argo Rollouts within a single namespace without requiring cluster permissions.
+** This mode provides security benefits by limiting the cluster scope of a single Argo Rollouts instance to a specific namespace.
+
+[NOTE]
+====
+{gitops-title} does not allow both modes of Argo Rollouts installation at the same time to prevent unintended privilege escalation.
```

Comment:
Reads a little like neither mode is allowed to prevent unintended privilege escalation. Maybe something like: "To prevent unintended privilege escalation, {gitops-title} allows only one mode of Argo Rollout installation at a time" instead.

Reply:
Fixed.